packages/data-stores/src/domain-suggestions/queries.ts

```diff
@@ -58,14 +51,15 @@ export function useGetDomainSuggestions(
 
 	return {
 		...result,
-		invalidateCache: () => queryClient.invalidateQueries( queryKey ),
+		invalidateCache: () =>
+			queryClient.invalidateQueries( [ 'domain-suggestions', search, searchOptions ] ),
```

Comment:
Thought this was more readable than trying to avoid DRY.

Reply:
I think you mean "to adhere DRY"? ;) 
The downside of this change is that it's now possible that someone would update the query key above but not this one. I don't feel particular strongly about this though, so please feel free to keep it this way :)